### PR TITLE
Update gast mirror URL

### PR DESCRIPTION
**Pull request by Saduf2019**
Updated the mirrored URL for gast_archive

Fix for #50777

### CODE REVIEW

File: tensorflow/workspace2.bzl [hunk 1]
```diff
@@ -429,13 +429,13 @@ def _tf_repositories():
     )
 
     tf_http_archive(
-        name = "gast_archive",
+        name = "F",
```
Comment (mihaimaruseac) [inline tensorflow/workspace2.bzl:432]: This is not right

File: configure.py [hunk 2]
```diff
@@ -50,7 +50,7 @@
 _TF_BAZELRC = ''
 _TF_CURRENT_BAZEL_VERSION = None
 _TF_MIN_BAZEL_VERSION = '3.7.2'
-_TF_MAX_BAZEL_VERSION = '3.99.0'
+_TF_MAX_BAZEL_VERSION = '4.1.0'
```
Comment (mihaimaruseac) [inline configure.py:53]: This is not related to this change

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: configure.py (hunk 2 above)
Comment (mihaimaruseac) [reply configure.py:53]:
```suggestion
_TF_MAX_BAZEL_VERSION = '3.99.0'
```

File: tensorflow/workspace2.bzl (hunk 1 above)
Comment (mihaimaruseac) [reply tensorflow/workspace2.bzl:432]:
```suggestion
        name = "gast_archive",
```

**Review by mihaimaruseac — APPROVED** [review]

**Comment by 8bitmp3** [comment]
@mihaimaruseac @lamberta @MarkDaoust 

Should we update the min. bazel version for v2.4 here https://www.tensorflow.org/install/source ? See https://github.com/tensorflow/tensorflow/issues/50966

Thanks @Saduf2019

**Comment by MarkDaoust** [comment]
>  min. bazel version for v2.4 here

I don't understand the question.

**Comment by 8bitmp3** [comment]
Never mind - it says 3.1, so 4.1 or 3.99 should do the job 👍
